File: CodeForces/Java/CF_131A_cAPS_lOCK.java
// Problem: 131A. cAPS lOCK
// Link: https://codeforces.com/problemset/problem/131/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    String word = scan.next();
    scan.close();

    boolean changeCase = true;
    for (int i = 1; i < word.length(); i++) {
        if (Character.isLowerCase(word.charAt(i))) {
            changeCase = false;
            break;
        }
    }

    if (changeCase) {
        StringBuilder newWord = new StringBuilder();
        for (int i = 0; i < word.length(); i++) {
            char c = word.charAt(i);
            if (Character.isUpperCase(c)) {
                newWord.append(Character.toLowerCase(c));
            } else {
                newWord.append(Character.toUpperCase(c));
            }
        }
        System.out.println(newWord.toString());
    } else {
        System.out.println(word);
    }
  }
}

File: CodeForces/Python/CF_131A_cAPS_lOCK.py
# Problem: 131A. cAPS lOCK
# Link: https://codeforces.com/problemset/problem/131/A

word = input()

change_case = True
for i in range(1, len(word)):
    if word[i].islower():
        change_case = False
        break

if change_case:
    new_word = ""
    for c in word:
        if c.isupper():
            new_word += c.lower()
        else:
            new_word += c.upper()
    print(new_word)
else:
    print(word)
